config/routes.rb

```diff
@@ -31,6 +31,7 @@
       put  "messages/:id" => "messages#update"
       put  "messages/:id/mark_read" => "messages#mark_read"
 
+      get  "offers/ggv_order_offer" => "offers#ggv_order_offer"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.